github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -20,12 +20,9 @@ postsubmits:
               - "/usr/local/bin/runner.sh"
               - "/bin/bash"
               - "-c"
-              - >-
-                cat $DOCKER_PASSWORD | docker login --username $(<$DOCKER_USER)
-                --password-stdin && cd images &&
-                source ./publish_image.sh
-                image_tag="$(get_image_tag)"
-                bazelisk run --define tag=${image_tag} //external-plugins/rehearse/plugin:push
+              - |
+                cat $DOCKER_PASSWORD | docker login --username $(<$DOCKER_USER) --password-stdin
+                bazelisk run --workspace_status_command hack/print-workspace-status.sh //external-plugins/rehearse/plugin:push
```

Comment:
the workspace status should go into a `.bazelrc` file to be consistent.

Here what we have for instance in kubevirt:

```
build --stamp --workspace_status_command=./hack/print-workspace-status.sh --host_force_python=PY3
```

Reply:
Done, PTAL